index.css

```diff
@@ -186,3 +186,11 @@ footer h1 {
 footer a:hover {
     color: #f7cd55;
 }
+.reset{
+    text-align: center;
+    height: 50px;
+        width: 50px;
```

Comment:
add this with the other pull request because it references the other file. Just delete this pull request and add more changes to the other one.